src/benchmarks/config_reader.x

```diff
@@ -0,0 +1,15623 @@
+# 1 "config_reader.cpp"
```

Comment:
This file seems to have been mistakenly added to the patch and should be removed.